plover/oslayer/osxkeyboardcontrol.py

```diff
@@ -232,6 +234,13 @@ def callback(proxy, event_type, event, reference):
             # Don't pass on meta events meant for this event tap.
             is_unexpected_event = event_type not in self._KEYBOARD_EVENTS
             if is_unexpected_event:
+                if event_type == kCGEventTapDisabledByTimeout:
+                    # Re-enable the tap and hope we act faster next time
+                    CGEventTapEnable(self._tap, True)
+                    plover.log.warning(
+                        "Keyboard event tap was disabled by timeout. " +
```

Comment:
OK, I ended up triggering this today. It worked out OK, but I might want a newline after the "disabled by timeout" bit.